bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/provider/ShellyTranslationProvider.java

```diff
@@ -10,40 +10,38 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  */
-package org.openhab.binding.shelly.internal.util;
+package org.openhab.binding.shelly.internal.provider;
 
 import java.util.Locale;
 
-import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.eclipse.jdt.annotation.Nullable;
 import org.openhab.core.i18n.LocaleProvider;
 import org.openhab.core.i18n.TranslationProvider;
 import org.osgi.framework.Bundle;
+import org.osgi.framework.FrameworkUtil;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
 
 /**
  * {@link ShellyTranslationProvider} provides i18n message lookup
  *
  * @author Markus Michels - Initial contribution
  */
-@NonNullByDefault
+@Component(service = ShellyTranslationProvider.class)
```

Comment:
Did you remove the `@NonNullByDefault` accidentally?

Reply:
obviously yes, fixed